Dockerfile

```
@@ -107,6 +108,10 @@ COPY --from=builder /app/public ./public
 RUN mkdir .next
 RUN chown nextjs:nodejs .next
 
+# Set the correct permissions for the tina cache folder
+RUN mkdir -p /app/tina
+RUN chown nextjs:nodejs /app/tina
```

```suggestion
RUN mkdir -p /app/tina && chown -R nextjs:nodejs /app/tina
```

I think we just need to give Read permission

---

Tina is writing cache files to this directory, so I'm pretty sure it should have write permissions as well. 


The exception being thrown in the app service logs says that the error occurs when Next JS attempts a `mkdir` operation in `/app/tina`

---

ok makes sense!